pkg/services/folder/folderimpl/folder.go

```diff
@@ -555,7 +557,9 @@ func (s *Service) Create(ctx context.Context, cmd *folder.CreateFolderCommand) (
 	dashFolder := dashboards.NewDashboardFolder(cmd.Title)
 	dashFolder.OrgID = cmd.OrgID
 
-	if s.features.IsEnabled(ctx, featuremgmt.FlagNestedFolders) && cmd.ParentUID != "" {
+	// TODO: This feature flag check causes issue for provisioning: folder is created without FolderUID in dashboards table
+	// Then provisioning getOrCreateFolderByTitle fails to find the folder and tries to create it again even thought it already exists
```

Comment:
@papagian Sofia, I need your help with deciding if it's the right thing to do. Suggest to discuss it offline.